include/openssl/e_os2.h

```diff
@@ -258,29 +258,6 @@ typedef unsigned __int64 uint64_t;
 #  include <stdint.h>
```

Comment:
good catch!   updated commit pushed.  (BTW it's probably not wrong since UEFI is a special embedded platform...)